src/parser/mcmt/mcmt_state.cpp

```diff
@@ -172,6 +181,8 @@ bool mcmt_state::is_declared(std::string id, mcmt_object type) const {
   case MCMT_OBJECT_LAST:
     // Always no op
     return false;
+  case MCMT_PROCESS_TYPE:
```

Comment:
It seems like MCMT_OBJECT_LAST should be the last in the enumeration. There might be dependencies elsewhere in code.